pom.xml

```diff
@@ -182,6 +180,8 @@
         <common-compress.version>1.14</common-compress.version>
         <owasp-html-sanitizer.version>20171016.1</owasp-html-sanitizer.version>
         <!--START If these bundles are used elsewhere in DDF, the versions MUST exactly match the versions used by the CXF and Camel features.-->
+        <javax.annotation-api.version>1.3</javax.annotation-api.version>
```

Comment:
@emmberk properties moved per your request 👍

Reply:
Looks great. Thank you!